### PR TITLE
Adds French and Italian mnemonic word lists (BIP39)

### PR DESCRIPTION
**Pull request by thecodefactory**
Requires libbitcoin/libbitcoin#683

### CODE REVIEW

File: model/generate.xml [hunk 1]
```diff
@@ -407,13 +407,13 @@
   </command>
   
   <command symbol="mnemonic-new" output="string" multipleX="true" category="WALLET" description="Create a mnemonic seed (BIP39) from entropy. WARNING: mnemonic should be created from properly generated entropy.">
-    <option name="language" type="language" description="The language identifier of the mnemonic dictionary to use. Options are 'en', 'es', 'ja', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'en'." />
+    <option name="language" type="language" description="The language identifier of the mnemonic dictionary to use. Options are 'en', 'es', 'ja', 'it', 'fr', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'en'." />
```
Comment (evoskuil) [inline model/generate.xml]: You should add comments to the XML as a reminder to update the command help in the wiki.

File: model/generate.xml [hunk 2]
```diff
@@ -407,13 +407,13 @@
...
     <argument name="SEED" stdin="true" type="base16" description="The Base16 entropy from which the mnemonic is created. The length must be evenly divisible by 32 bits. If not specified the entropy is read from STDIN." />
     <define name="BX_EC_MNEMONIC_NEW_INVALID_ENTROPY" value="The seed length in bytes is not evenly divisible by 32 bits." />
   </command>
 
   <command symbol="mnemonic-to-seed" output="base16" category="WALLET" description="Convert a mnemonic seed (BIP39) to its numeric representation.">
-    <option name="language" type="language" description="The language identifier of the dictionary of the mnemonic. Options are 'en', 'es', 'ja', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'any'." />
+    <option name="language" type="language" description="The language identifier of the mnemonic dictionary to use. Options are 'en', 'es', 'ja', 'it', 'fr', 'zh_Hans', 'zh_Hant' and 'any', defaults to 'en'." />
```
Comment (evoskuil) [inline model/generate.xml]: The wording of this option description is purposely distinct from `mnemonic-new`. The default should be `any` and the purpose is to restrict the command to using the specified language to interpret the mnemonic, so the option indicates the dictionary "of the mnemonic".

File: model/generate.xml (hunk 1 above)
Comment (thecodefactory) [inline model/generate.xml]: What's the target date for wiki updates?  I could update it sooner than later, I suppose.  What's the process for distinguishing between v2/v3 btw?

**Comment by thecodefactory** [comment]
Updated.

**Comment by thecodefactory** [comment]
@evoskuil Anything outstanding here?

**Comment by evoskuil** [comment]
Thanks!

**Pull request merged**